src/index.ts

```diff
@@ -85,6 +85,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     configurations.pollingInterval = MIN_POLLING_INTERVAL
   }
 
+  if (configurations.streamEnabled) {
+    try {
+      const { Platform } = require("react-native");
+      if  (Platform.OS === 'android') {
+        console.warn("SDKCODE:1007 workaround applied - Android React Native detected - streaming will be disabled and polling enabled")
+        configurations.pollingEnabled = true
+        configurations.streamEnabled = false
+      }
+    } catch (e) {
+      // ignore
```

Comment:
should we log this, or this is what we would hit if its a non-react-native env and we just want to ignore?

Reply:
yeah it's pretty much a non-op for non react native, most solutions i've found require this method of using a catch block